src/bootstrap/util.rs

```diff
@@ -32,6 +32,11 @@ pub fn is_dylib(name: &str) -> bool {
     name.ends_with(".dylib") || name.ends_with(".so") || name.ends_with(".dll")
 }
 
+/// Returns `true` if the file name given looks like a debug info file
+pub fn is_debug_info(name: &str) -> bool {
+    name.ends_with(".pdb")
```

Comment:
I'm now wondering if this is also necessary for similar cases with split debuginfo on linux/macOS; could you add a comment here that "we may need to consider linux/macOS split debuginfo here too"?